src/font.ts

```diff
@@ -261,44 +261,6 @@ export class Font {
     }
   }
 
-  /**
-   * Customize this field to specify a different CDN for delivering web fonts.
-   * Discussion on GDPR concerns:
-   * https://www.jsdelivr.com/blog/how-the-german-courts-ruling-on-google-fonts-affects-jsdelivr-and-why-it-is-safe-to-use/
-   *
-   * You can also self host, and specify your own server URL here.
-   */
-  static WEB_FONT_HOST = 'https://cdn.jsdelivr.net/npm/vexflow-fonts@1.0.6/';
-
-  /**
-   * These font files will be loaded from the CDN specified by `Font.WEB_FONT_HOST` when
-   * `await Font.loadWebFonts()` is called. Customize this field to specify a different
-   * set of fonts to load. See: `Font.loadWebFonts()`.
-   */
-  static WEB_FONT_FILES: Record<string /* fontName */, string /* fontPath */> = {
-    Academico: 'academico/Academico_0.902.woff2',
-    Bravura: 'bravura/Bravura_1.392.woff2',
-    BravuraText: 'bravura/BravuraText_1.393.woff2',
-    GonvilleSmufl: 'gonvillesmufl/GonvilleSmufl_1.100.woff2',
-    Gootville: 'gootville/Gootville_1.3.woff2',
-    GootvilleText: 'gootville/GootvilleText_1.2.woff2',
-    Leland: 'leland/Leland_0.75.woff2',
-    LelandText: 'leland/LelandText_0.75.woff2',
-    Petaluma: 'petaluma/Petaluma_1.065.woff2',
-    'Petaluma Script': 'petaluma/PetalumaScript_1.10_FS.woff2',
-    MuseJazz: 'musejazz/MuseJazz_1.0.woff2',
-    MuseJazzText: 'musejazz/MuseJazzText_1.0.woff2',
-    'Roboto Slab': 'robotoslab/RobotoSlab-Medium_2.001.woff2',
-    FinaleAsh: 'finale/FinaleAsh_1.7.woff2',
-    FinaleAshText: 'finale/FinaleAshText_1.3.woff2',
-    FinaleJazz: 'finale/FinaleJazz_1.9.woff2',
-    FinaleJazzText: 'finale/FinaleJazzText_1.3.woff2',
-    FinaleBroadway: 'finale/FinaleBroadway_1.4.woff2',
-    FinaleBroadwayText: 'finale/FinaleBroadwayText_1.1.woff2',
-    FinaleMaestro: 'finale/FinaleMaestro_2.7.woff2',
-    FinaleMaestroText: 'finale/FinaleMaestroText-Regular_1.6.woff2',
-  };
-
```

Comment:
I am curious why the ability to load web fonts from CDN was deleted?

I thought you wanted to have a vexflow build that bundles zero fonts. To do that, we should have easy APIs for people to call to load their desired font. I'm in the middle of improving this font loading, but I don't think it should be removed altogether.

Reply:
That was just a check to see that it was not usted. I will revert it.

Reply:
? :-) You could have asked haha. I appreciate that you are going through and removing deprecated & old stuff that no may longer apply, but:

1) this deletion isn't related to this PR.

2) isn't it the job of our library to make it easier for developers to use SMuFL fonts? If we remove all the font loading code from the library, every developer will have to figure it out for themselves. You've spent a lot of time on this code base so the idea of preloading a SMuFL font seems easy, but I think there are lots of developers who would appreciate having an easy API call to load a web font (even if they end up rolling their own custom solution later on).

Reply:
@ronyeh I left that by mistake after a try. I think though that leaving loadWebFont is enough, the user should be able to decide on cdn and font. Anyway as you said, no part of this. :)

Reply:
reverted